packages/common-lib/src/index.tsx

```diff
@@ -28,6 +28,9 @@ import * as teacherRegistryService from './services/teacherRegistryService'
 import * as classRegistryService from './services/classRegistryService'
 import * as attendanceRegistryService from './services/attendanceRegistryService'
 import * as studentRegistryService from './services/studentRegistryService'
+import * as worksheetRegistryService from './services/worksheetRegistryService'
```

Comment:
can we move the service code to respective module?

Reply:
this approach suggested by @ChakshuGautam

Reply:
is that service used in across modules? if not then it must be part of the respective module. worksheetService in worksheet module.

Reply:
Hey @sagarkoshti1990 can you point me to the discussion where I suggested this. I think I might have miscommunicated here.